emr-user-role-mapper-application/pom.xml

```diff
@@ -106,7 +106,12 @@
             <artifactId>aws-java-sdk-sts</artifactId>
             <version>${aws-java-sdk.version}</version>
         </dependency>
-        <dependency>
+      <dependency>
```

Comment:
looks like the indentation is incorrect. pls fix

Reply:
Done.